### PR TITLE
.travis.yml

### PR DESCRIPTION
**Pull request by dnsl48**
- fix versions
- add postgres
- ~remove~ fix coverage (it hasn't been working for a while)
- add php 7.3

### CODE REVIEW

**Comment by dnsl48** [comment]
because we don't have it set up on this repository. I would guess it's just came into this travis.yml as a copy-paste from another repo in the first place. I don't see coverage.xml anywhere.

**Comment by robbieaverill** [comment]
Ahh, you're right - we just need to add the whitelist to phpunit.xml.dist:

```diff
<?xml version="1.0" encoding="UTF-8"?>
<!--
Standard module phpunit configuration.
Requires PHPUnit ^5.7
 -->
<phpunit bootstrap="vendor/silverstripe/framework/tests/bootstrap.php" colors="true">
	<testsuite name="Default">
		<directory>tests/php</directory>
    </testsuite>
+    <filter>
+        <whitelist addUncoveredFilesFromWhitelist="true">
+            <directory suffix=".php">src/</directory>
+            <exclude>
+                <directory suffix=".php">tests/php/</directory>
+            </exclude>
+        </whitelist>
+    </filter>
</phpunit>
```

**Comment by dnsl48** [comment]
sweet, now we have codecov here :champagne:

File: .travis.yml [hunk 1]
```diff
@@ -4,33 +4,42 @@ sudo: false
...
     - php: 5.6
-      env: DB=MYSQL PHPUNIT_TEST=1 PHPCS_TEST=1
+      env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.0
       env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL PDO=1 PHPUNIT_COVERAGE_TEST=1
+      env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.2
+      env: DB=MYSQL PHPUNIT_TEST=1  PHPCS_TEST=1 PHPUNIT_COVERAGE_TEST=1
```
Comment (robbieaverill) [inline .travis.yml]: Having both PHPUNIT_TEST and PHPUNIT_COVERAGE_TEST will run the test suite twice, you can remove PHPUNIT_TEST

**Comment by dnsl48** [comment]
done. Also fixed the PDO test run

**Pull request merged**